README.md

```diff
@@ -71,6 +71,7 @@ Use "yast [command] --help" for more information about a command.
 ## 📝 NOTE
 1. The current supported target requires VPN to get results. Please make sure that you are connected to a **VPN Server** before searching or you may not get the results. We personally recommend using CloudFlare Warp. You can [download](https://1.1.1.1/) it here.
 2. The Project is still in development stage. So, you might see undesired behaviour while using yast. Please file an issue if you experience any bugs or undesired behaviour. 
+3. Currently `wsl2` does not support yast, because `yast` works on targets that can be accessed by a VPN only and `wsl2` does not have VPN support yet.
```

Comment:
Delete the whole line and replace it with this. 

yast doesnot support **WSL2**, because the current targets available in yast, only work with **VPN** and it's very difficult to establish a **VPN** connection on **WSL2**.

Reply:
Please check, I didn't know how to make changes to the existing PRs via VS Code, so I made changes directly from github. Let me know please if I did something wrong.

Reply:
Its not reflecting on the changes. Can you look into that. Also just make a commit and  
do `git push` on the existing branch you pushed the previous changes with. They get automatically reflected here.

Reply:
Also, can you copy the content of `docs/docs.md` to end of the read me under the heading `System Design`. Rename docs.md -> design.md

Reply:
We can merge after these changes.